docs/use_cases/model_applications/convection_allowing_models/Point2Grid_ObsOnly_obsLSR_PracticallyPerfect.py

```diff
@@ -0,0 +1,141 @@
+===========================================================================================================================
+
+Point2Grid_ObsOnly_obsLSR_PracticallyPerfect.conf
+
+"""
+##############################################################################
+# Scientific Objective
+# --------------------
+#
+# To use storm reports as observations to calculate 
+# Practically Perfect probabilities.
+#
+
+##############################################################################
+# Datasets
+# --------
+#
+# Relevant information about the datasets that would be beneficial include:
+# 
+#  * Observation dataset: Local Storm Reports
+#
+
+##############################################################################
+# METplus Components
+# ------------------
+#
+# This use case runs ASCII2NC to get the storm reports in netcdf format, runs
+# Point2Grid to get those netcdf observations onto a grid, runs RegridDataPlane
+# to use that gridded data as a mask to calculate probabilities 
+#
+
+##############################################################################
+# METplus Workflow
+# ----------------
+#
+# The following tools are used for each run time:
+#
+# ASCII2NC > Point2Grid > RegridDataPlane 
+#
+# This example runs on a single time/file at a time. Each storm report is 
+# assumed to have no more than 24 hours of data inside 
+#
+# Run times:
+#
+# | 2020-02-05
+#
+
+##############################################################################
+# METplus Configuration
+# ---------------------
+#
+# METplus first loads all of the configuration files found in parm/metplus_config,
+# then it loads any configuration files passed to METplus via the command line
+# with the -c option, i.e. -c parm/use_cases/model_applications/convection_allowing_models/Point2Grid_ObsOnly_obsLSR_PracticallyPerfect.conf
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/model_applications/convection_allowing_models/Point2Grid_ObsOnly_obsLSR_PracticallyPerfect.conf
+#
+
+##############################################################################
+# MET Configuration
+# ---------------------
+#
+# METplus sets environment variables based on the values in the METplus configuration file.
+# These variables are referenced in the MET configuration file. **YOU SHOULD NOT SET ANY OF THESE ENVIRONMENT VARIABLES YOURSELF! THEY WILL BE OVERWRITTEN BY METPLUS WHEN IT CALLS THE MET TOOLS!** If there is a setting in the MET configuration file that is not controlled by an environment variable, you can add additional environment variables to be set only within the METplus environment using the [user_env_vars] section of the METplus configuration files. See the 'User Defined Config' section on the 'System Configuration' page of the METplus User's Guide for more information.
+#
+# .. highlight:: bash
+# .. literalinclude:: ../../../../parm/use_cases/
```

Comment:
This will cause an error in Sphinx because it is a literal include of a directory, not a file path.

Reply:
Added Ascii2NcWrapped. Checked in outside of github.

Reply:
Do we need to rename these?